src/test/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintHandlerTest.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.cmc.claimstore.documents;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.cmc.claimstore.events.DocumentReadyToPrintEvent;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.reform.sendletter.api.Document;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BulkPrintHandlerTest {
+
+    @Mock
+    private BulkPrintService bulkPrintService;
+
+    @Test
+    public void notifyStaff() {
+        //given
+        BulkPrintHandler bulkPrintHandler = new BulkPrintHandler(bulkPrintService);
+        Claim claim = SampleClaim.getDefault();
+        Map<String, Object> pinContents = new HashMap<>();
+        Document defendantLetterDocument = new Document("pinTemplate", pinContents);
+        Map<String, Object> claimContents = new HashMap<>();
```

Comment:
Can be inlined

Reply:
I thought of same, however left extracted just to know what is this empty hash map